Makefile

```diff
@@ -124,9 +132,19 @@ $(BINDIR)/informer-gen:
 
 $(BINDIR)/dep:
 	curl -sL -o $@ $(DEP_URL)
-	echo "$(DEP_HASH)  $@" | $(SHASUM)
+	echo "$(DEP_HASH) $@" | $(SHASUM)
 	chmod +x $@
 
+$(BINDIR)/node:
+	curl -sL -o $(BINDIR)/$(NODE_NAME).$(TAR_EXT) $(NODE_URL)
+	echo "$(NODE_HASH) $(BINDIR)/$(NODE_NAME).$(TAR_EXT)" | $(SHASUM)
```

Comment:
Add extra space between `$(NODE_HASH)` and `$(BINDIR)/$(NODE_NAME).$(TAR_EXT)`. This is needed to make it work on MacOS.